pkg/cloudproviders/gcp/auth/cred_manager_impl_test.go

```diff
@@ -116,23 +116,23 @@ func TestCredentialManager(t *testing.T) {
 		},
 	}
 
-	for name, c := range cases {
-		c := c
+	for name, tc := range cases {
+		tc := tc
 		t.Run(name, func(t *testing.T) {
 			t.Parallel()
 			k8sClient := fake.NewSimpleClientset()
 			manager := newCredentialsManagerImpl(k8sClient, namespace, secretName, func() {})
 			manager.Start()
 			defer manager.Stop()
 
-			err := c.setupFn(k8sClient)
+			err := tc.setupFn(k8sClient)
 			require.NoError(t, err)
 
 			// Assert that the secret data has been updated.
-			assert.EventuallyWithT(t, func(collect *assert.CollectT) {
+			assert.EventuallyWithT(t, func(c *assert.CollectT) {
```

Comment:
How about using t instead of c so diff will be minimal as no rename will be necessary. (credits to @0x656b694d )
```suggestion
			assert.EventuallyWithT(t, func(t *assert.CollectT) {
```